test/unit/utils.py

```diff
@@ -58,3 +86,13 @@ def assertRaisesRegexp(expected_exception, expected_regexp,
         else:
             excName = str(expected_exception)
         raise unittest.TestCase.failureException("%s not raised" % excName)
+
+
+def get_random_unicode(length):
+    """Get a random Unicode string."""
+    u_str = u''
+    for _ in range(length):
+        u_str += unichr(random.choice(range(0x0370, 0x0377) +
+                                      range(0x0384, 0x038A)))
+
```

Comment:
The approach above will be fairly painful for larger strings. Why not

```python
UNICODE_CHARS = [unichr(i) for i in itertools.chain(xrange(0x0370, 0x377), xrange(0x0384, 0x038a)]

def get_random_unicode(length):
    return u''.join(random.choice(UNICODE_CHARS) for _ in xrange(length))
```

By the way, are you sure about those ranges? 0x0370 to 0x0377 and 0x0384 to 0x038a only yields 13 characters?